src/components/CommentCard.jsx

```diff
@@ -1,18 +1,45 @@
-import moment from "moment"
+import { deleteCommentFromArticle } from "./api"
+import { useParams } from "react-router"
+import { useState, useContext } from "react"
+import { UserLoginContext } from "../contexts/UserLogin"
 
-function CommentCard({comment}) {
+function CommentCard({comment, setRefresh, isLoading, setIsLoading}) {
+
+    const {comment_id} = comment
+    const [isDeleteButtonDisabled, setIsDeleteButtonDisabled] = useState(false)
+    const { loggedUser } = useContext(UserLoginContext)
+    
 
     const dateAndTime = comment.created_at.split("T")
     const time = dateAndTime[1].split(":")
     const date = dateAndTime[0].split("-")
+
+    function disableDeleteButton() {
+        setIsDeleteButtonDisabled(true)
+    }
+
+    function handleDelete(event) {
+        event.preventDefault()
+        disableDeleteButton()
```

Comment:
You could just call setIsDeleteButtonDisabled(true) here rather than extract it into a func.